app/components/primer/octicon_symbol_component.rb

```diff
@@ -61,7 +60,7 @@ def initialize(symbol:, size: Primer::OcticonComponent::SIZE_DEFAULT)
       end
 
       def call
-        render(Primer::BaseComponent.new(**@system_arguments)) { @icon.path.html_safe } # rubocop:disable Rails/OutputSafety
+        content_tag(:symbol, @icon.path.html_safe, @system_arguments) # rubocop:disable Rails/OutputSafety
```

Comment:
As we're not sending these arguments into Classify, might we call them `tag_arguments`?